src/filing/submission/router.jsx

```diff
@@ -97,37 +104,45 @@ export class SubmissionRouter extends Component {
     const splat = this.props.match.params.splat
     const latest = this.getLatestPage()
 
-    this.renderChildren = true
-
     if (!splat) {
       return this.replaceHistory(latest)
     }
 
     if (submissionRoutes.indexOf(splat) === -1) {
-      return this.goToAppHome()
+      return this.replaceHistory(latest)
     }
 
     if (code <= VALIDATING)
-      if (splat === 'upload') return this.forceUpdate()
+      if (splat === 'upload') return this.update()
       else return this.replaceHistory('upload')
 
-    if (code >= VALIDATING && code <= VALIDATED) {
-      if (splat === latest) return this.forceUpdate()
+    if (code > VALIDATING && code <= VALIDATED) {
+      if (splat === latest) return this.update()
       else if (
         submissionRoutes.indexOf(splat) > submissionRoutes.indexOf(latest)
       ) {
         return this.replaceHistory(latest)
       }
     }
 
+    return this.update()
+  }
+
+  update(){
+    this.renderChildren = true
     return this.forceUpdate()
   }
 
   render() {
-    const { submission, match: {params} } = this.props
+    const { submission, lei, match: {params} } = this.props
+    const { code } = submission.status
+
     if (
-      submission.status.code === UNINITIALIZED ||
+      code === UNINITIALIZED ||
+      (code < VALIDATED && code !== NO_MACRO_EDITS && params.splat === 'submission') ||
```

Comment:
Just trying to understand what this scenario is...

User is trying to access the `signing` page but have not reached that part of the process.  What would be loading here?